crest/Assets/Crest/Crest/Shaders/Resources/ShapeCombine.shader

```diff
@@ -20,6 +20,7 @@ Shader "Hidden/Crest/Simulation/Combine Animated Wave LODs"
 			CGPROGRAM
 			#pragma vertex Vert
 			#pragma fragment Frag
+			#pragma target 3.5
```

Comment:
Does this need to be added to the other pass below?

Reply:
Ah yes good point

Reply:
done